site/content/docs/main/api-types/restore.md

```diff
@@ -73,6 +73,55 @@ spec:
   # to restore from. If specified, and BackupName is empty, Velero will
   # restore from the most recent successful backup created from this schedule.
   scheduleName: my-scheduled-backup-name
+  # Actions to perform during or post restore. The only hook currently supported is
+  # adding an init container to a pod before it can be restored. Optional.
+  hooks:
+    # Array of hooks that are applicable to specific resources. Optional.
+    resources:
+    # Name is the name of this hook.
+    - name: restore-hook-1
+      # Array of namespaces to which this hook applies. If unspecified, the hook applies to all
+      # namespaces. Optional.
+      includedNamespaces:
+      - ns1
+      # Array of namespaces to which this hook does not apply. Optional.
+      excludedNamespaces:
+      - ns3
+      # Array of resources to which this hook applies. The only resource supported at this time is
+      # pods.
+      includedResources:
+      - pods
+      # Array of resources to which this hook does not apply. Optional.
+      excludedResources: []
+      # This hook only applies to objects matching this label selector. Optional.
+      labelSelector:
+        matchLabels:
+          app: velero
+          component: server
+      # An array of hooks to run during or after restores. Currently only "init" hooks are supported.
+      postHooks:
+      # The type of the hook. This must be "init".
+      - init:
+          # An array of container specs to be added as init containers to pods to which this hook applies to.
+          initContainers:.
+          - name: restore-hook-init1
+            image: alpine:latest
+            volumeMounts:
+            - mountPath: /restores/pvc1-vm
+              name: pvc1-vm
```

Comment:
These volume names come from the original pod spec that is being restored, right? Would it be worth adding an comment above to explain that?

Reply:
added comments.